rg3d-ui/src/file_browser.rs

```diff
@@ -742,9 +858,93 @@ impl<M: MessageData, C: Control<M, C>> FileBrowserBuilder<M, C> {
             scroll_viewer,
             root: self.root,
             file_name,
+            watcher: Rc::new(cell::Cell::new(None)),
         };
+        let watcher = browser.watcher.clone();
+        let filebrowser_node = UINode::FileBrowser(browser);
+        let node = ctx.add_node(filebrowser_node);
+        watcher.replace(setup_filebrowser_fs_watcher(
+            ctx.ui.sender(),
+            node,
+            the_path,
+        ));
+        node
+    }
+}
 
-        ctx.add_node(UINode::FileBrowser(browser))
+fn setup_filebrowser_fs_watcher<M: MessageData, C: Control<M, C>>(
+    ui_sender: mpsc::Sender<UiMessage<M, C>>,
+    filebrowser_widget_handle: Handle<UINode<M, C>>,
+    the_path: PathBuf,
+) -> Option<(notify::RecommendedWatcher, thread::JoinHandle<()>)> {
+    let (tx, rx) = mpsc::channel();
+    match notify::watcher(tx, time::Duration::from_secs(1)) {
+        Ok(mut watcher) => {
+            #[allow(clippy::while_let_loop)]
+            let watcher_conversion_thread = std::thread::spawn(move || loop {
+                println!("Waiting for FS Watcher Event....");
```

Comment:
Could you please remove these `println`s? I guess these left over from the debugging?